### PR TITLE
EB-10680 Fix round tax

### PR DESCRIPTION
**Pull request by aallier**

### CODE REVIEW

File: Tests/Service/OgoneFileBuilderTest.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+//require '../../vendor/autoload.php';
```
Comment (lepiaf) [inline Tests/Service/OgoneFileBuilderTest.php]: remove

File: Service/OgoneFileBuilder.php [hunk 2]
```diff
@@ -63,11 +63,11 @@ public function buildInv($orderId, $clientId, $clientRef, $legalCommitment, $ali
             $price     = $quantity * $unitPrice;
             $articlesLines[$k] = $this->createDetailLineArray($quantity, $id, $name, $unitPrice, $vat, $price);
             $amountTaxExcluded += $price; //tax excluded
-            $amountValueAddedTax += round($price * $vat);
+            $amountValueAddedTax += $price * $vat;
             $nbArticles++;
         }
 
-        $amountTaxIncluded = $amountTaxExcluded + $amountValueAddedTax; //amount taxes included
+        $amountTaxIncluded = $amountTaxExcluded + round($amountValueAddedTax, 2); //amount taxes included
```
Comment (ch3ric) [inline Service/OgoneFileBuilder.php]: this amount has been multiplied by 100 above, so round without precision

File: Tests/Service/OgoneFileBuilderTest.php [hunk 3]
```diff
@@ -135,7 +135,7 @@ public function testValidRound()
     {
         $expectedTextContent = "OHL;ETSCPC;anglet64600;;userapi64600;
 OHF;FILEorder_id25;MTR;SAS;1;
-INV;EUR;;;;transactionId;LEGAL;;payId;SAS;;;;ETSCPC;;1;aliasGSP;azerty;;;1700065264;;;;;;;;order_id25;;;4403;881;5284;
+INV;EUR;;;;transactionId;LEGAL;;payId;SAS;;;;ETSCPC;;1;aliasGSP;azerty;;;1700065264;;;;;;;;order_id25;;;4403;880.6;5283.6;
```
Comment (ch3ric) [inline Tests/Service/OgoneFileBuilderTest.php]: this should not change, do not send floats

Comment (ch3ric) [reply Tests/Service/OgoneFileBuilderTest.php]: please add another test case to reproduce your exact issue, but this test case should not change

**Pull request merged**